frontend/src/lib-components/molecules/FileDownloadButton.tsx

```diff
@@ -21,10 +25,23 @@ const DownloadButton = styled.button`
   text-decoration: none;
 `
 
+function getIcon(contentType: string): IconDefinition {
```

Comment:
`FileUpload.tsx` exports `fileIcon` function in `lib-components/molecules`

Reply:
oh, thanks 👍